### DISCUSSION TITLE
SourceMaps

### DISCUSSION THREAD

**Discussion by completewebco** [tailpress/tailpress#81] [Q&A]
Is there a way to display source maps rather than just css/app.css in web inspector?

**Comment by jeffreyvr** [comment]
If you're using Laravel Mix you could try:

https://laravel-mix.com/docs/6.0/api

```js
mix.postCss("resources/css/app.css", "css").sourceMaps();
```

Haven't tested this.

**Reply by completewebco** [reply]
Thanks, Jeffery! I am going to switch to the Tailwind CLI. Do you know if sourcemaps are included with this? I guess I could ask my question there and not here...

**Reply by jeffreyvr** [reply]
Not sure, but I don't think it is.

**Comment by completewebco** [comment]
Using Laravel Mix I was able to get source maps to work adding:

`mix.sourceMaps( true, 'source-map' );`
